src/vue/__tests__/utils.test.ts

```diff
@@ -41,7 +41,7 @@ describe("utils", () => {
       const options = { retry: false };
       const result = parseMutationArgs(options);
 
-      expect(result).toBe(options);
```

Comment:
We do not extend the original data anymore. We always create a new reactive object out of the arguments